torch_np/_normalizations.py

```diff
@@ -96,6 +96,17 @@ def maybe_normalize(arg, parm, return_on_failure=_sentinel):
             raise exc from None
 
 
+def wrap_tensors(result):
+    from ._ndarray import ndarray
+
+    if isinstance(result, torch.Tensor):
+        result = ndarray(result)
+    elif isinstance(result, (tuple, list)):
+        result = type(result)(ndarray(x) if isinstance(x, torch.Tensor) else x for x in result)
```

Comment:
I think this should be good enough in practice, but if not, this can more generally be written as:
```suggestion
        result = type(result)(wrap_tensors(x) x for x in result)
```

Reply:
In principle, yes. I don't think numpy has returns nested like that though. So feels like extra complexity, no?

Reply:
I'm fine with either.

Reply:
In fact this is useful in gh-93, so done over there.